.changeset/smart-owls-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nestjs-shopify/auth": minor
```

Comment:
Also I don't think this is a non-breaking change anymore. I'm okay with making this a `major` change as its quite a big change, and we _might_ still unknowingly break something.

Reply:
technically it should not be breaking, but yes, who knows :D 

if we make this a major change, should we then also require the latest shopify-sdk?

Reply:
That sounds good! We can bump the versions 👍